pyproject.toml

```diff
@@ -0,0 +1,53 @@
+[build-system]
+requires = ["setuptools >= 61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "sphinxcontrib-images"
+version = "0.9.4"
+requires-python = ">= 3.9"
+
+authors = [
+    {name = "Tomasz Czyż", email = "tomasz.czyz@gmail.com"},
+]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Environment :: Console",
+    "Environment :: Web Environment",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Topic :: Documentation",
+]
+dependencies = [
+    "requests>2.2,<3",
+    "setuptools",  # TODO: migrate off of pkg_resources
```

Comment:
This is now properly declared as a dependency until pkg_resources is not used here:

https://github.com/sphinx-contrib/images/blob/30b50b8920c82ebea6a64538cded11580428cf24/sphinxcontrib/images.py#L285-L288

It's no longer needed for namespace packages